react/src/Calendar/CalendarBase/utils.ts

```diff
@@ -4,7 +4,7 @@ import {
   startOfDay,
   subDays,
 } from 'date-fns'
-import range from 'lodash/range'
+import { range } from 'lodash'
```

Comment:
My knowledge of FE build tools is aged. Just curious, would this change affect how code splitting works? 🤔 

As `design-system` is a shared UI lib, it should definitely always aim to import as little as possible. In the old days, the change here would mean going from bundling only `range` to the whole `lodash` library I believe.

If that's the case, we should probably focus on fixing our Vite config instead.

Reply:
our vite build pipeline ignores lodash, so the treeshaking is up to the installer, but ya will still need to fix vite config eventually haha